src/editors/autocompleteEditor.js

```diff
@@ -126,11 +126,11 @@ class AutocompleteEditor extends HandsontableEditor {
         const autoColumnSize = this.getPlugin('autoColumnSize');
         let columnWidth = width;
 
-        if (autoColumnSize) {
-          const autoWidths = autoColumnSize.widths;
+        if (autoColumnSize && autoColumnSize.columnWidthsMap) {
+          const cachedWidth = autoColumnSize.getColumnWidth(col, void 0, false);
```

Comment:
Maybe it's possible to do it without using the API of the plugin? 

PS. I've some propositions for index mappers API (extra `getMap` function): https://github.com/handsontable/handsontable/commit/030e3eafabb2dcf6ae0731416b47bdf29e98a702. Maybe getting a map for a key and using it will be enough.

Reply:
What if `autoColumnSize` will change the map name? Using plugin API is safer and can be type checked by compilator.

Reply:
I know, it's not a perfect solution to have direct references between plugins, although, I think it's not a point of this refactor. In this case, I prefer to use public API instead of direct access to other else plugin's mapper.

Reply:
Okey 👍